sharness/t0021-service-init.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+test_description="Test init functionality"
+
+. lib/test-lib.sh
+test_ipfs_init
+
+test_expect_success "prerequisites" '
+    test_have_prereq IPFS
+'
+
+PEERS=/ip4/192.168.0.129/tcp/9196/ipfs/12D3KooWRN8KRjpyg9rsW2w7StbBRGper65psTZm68cjud9KAkaW,/ip4/192.168.0.129/tcp/9196/ipfs/12D3KooWPwrYNj7VficHw5qYidepMGA85756kYgMdNmRM9A1ZHjN
+
+test_expect_success "cluster-service init with --peers succeeds and fills peerstore" '
+    ipfs-cluster-service --config "test-config" init --peers $PEERS
+    [ -f "test-config/peerstore" ]
+    IFS=','
+    read -ra P <<< $PEERS
+    grep -q ${P[0]} "test-config/peerstore"
```

Comment:
This line is naught. The test result depends on the last command so whether this one passes or fails doesn't batter. Same on next test. Please create a file with the expected results and use `test_cmp` to compare the resulting peerstore with the pre-created file. `t0031-ct-add.sh` has examples.

Same problem in next test, in anything you need to `&&` tests.